rero/rero-ils
# indexing: update libraries mapping

* Adds analyzer on name field.
* Improves global analyzer.


How to test:

* `script/setup`
* http://localhost:5000/admin/records/libraries
* Find a term and check if the result is correct (value in fields name or description)

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
Co-Authored-by: Peter Weber <peter.weber@rero.ch>
